README.md

```diff
@@ -361,6 +362,43 @@ ctest --output-on-failure
 You will find compiled binaries in `build-output\` respective to their
 source directories.
 
+### Install
+
+By default `google-cloud-cpp` downloads and compiles all its dependencies.
+The default configuration disables the `install` target, because the version of
+the dependencies downloaded by `google-cloud-cpp` may conflict with the versions
+already installed in your system, or with the versions you want to use for
+development.
+
+To install `google-cloud-cpp` you must first install all its dependencies. Then
+you must configure `google-cloud-cpp` to find these dependencies, and install
+it.
+
+For example, if all the dependencies have installed the corresponding CMake
+support files, these commands will install `google-cloud-cpp`:
+
+```bash
+cmake -H. -Bbuild-output-for-install \
+    -DGOOGLE_CLOUD_CPP_DEPENDENCY_PROVIDER=package
+cmake --build build-output-for-install --target install
```

Comment:
I'd recommend removing an example in this README.md and instead link to INSTALL.md. I worry that users will copy and paste then execute the example to find it not work.

Reply:
Done. Thanks!